src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/CategoricalVariableCard.js

```diff
@@ -112,48 +124,53 @@ export default compose(
       variable: { bins },
     }) => getBinData(bins, dataBuckets)
   ),
-  withPropsOnChange((props, nextProps) =>
-  nextProps.variable.active_chart === 'survival' &&
-    (!isEqual(props.variable.bins, nextProps.variable.bins) ||
-    props.variable.active_chart !== nextProps.variable.active_chart ||
-    !isEqual(props.selectedSurvivalBins, nextProps.selectedSurvivalBins) ||
-    props.variable.setId !== nextProps.variable.setId ||
-    !isEqual(props.variable.customSurvivalPlots, nextProps.variable.customSurvivalPlots) ||
-    props.variable.isSurvivalCustom !== nextProps.variable.isSurvivalCustom),
+  withPropsOnChange(
+    (props, nextProps) =>
+      nextProps.variable.active_chart === 'survival' && (
```

Comment:
somehow mixing `&&` with `||` seems confusing to me (though it is pretty simple logic).
Could I suggest a `... && !(... && ...)` instead?

Reply:
I've fixed this in PRTL-2750